<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="maximumSize">
        <size>
         <width>631</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Строка поиска:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Найти следующий</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="createItemAct"/>
    <addaction name="openItemAct"/>
    <addaction name="saveItemAct"/>
    <addaction name="saveAsItemAct"/>
    <addaction name="mergeAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAct"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Записи</string>
    </property>
    <addaction name="addItemAct"/>
    <addaction name="deleteItemAct"/>
    <addaction name="editItemAct"/>
    <addaction name="separator"/>
    <addaction name="findItemAct"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>27</width>
     <height>27</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createItemAct"/>
   <addaction name="openItemAct"/>
   <addaction name="saveItemAct"/>
   <addaction name="saveAsItemAct"/>
   <addaction name="mergeAct"/>
   <addaction name="addItemAct"/>
   <addaction name="deleteItemAct"/>
   <addaction name="editItemAct"/>
   <addaction name="findItemAct"/>
  </widget>
  <action name="createItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-new-file-50.png</normaloff>:/rec/img/icons8-new-file-50.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-opened-folder-50.png</normaloff>:/rec/img/icons8-opened-folder-50.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-save-50.png</normaloff>:/rec/img/icons8-save-50.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-save-as-50.png</normaloff>:/rec/img/icons8-save-as-50.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="mergeAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-merge-48.png</normaloff>:/rec/img/icons8-merge-48.png</iconset>
   </property>
   <property name="text">
    <string>Объединить</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="addItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-add-new-50.png</normaloff>:/rec/img/icons8-add-new-50.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="deleteItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-remove-50.png</normaloff>:/rec/img/icons8-remove-50.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="editItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-pencil-50.png</normaloff>:/rec/img/icons8-pencil-50.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="findItemAct">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/rec/img/icons8-search-50.png</normaloff>:/rec/img/icons8-search-50.png</iconset>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
